docs/tables.rst

```diff
@@ -43,6 +43,7 @@ Creation
 
     Table.__init__
     Table.empty
+    Table.from_rows
```

Comment:
.from_rows is deprecated, I don't think we should add it to the docs

Reply:
I will make the necessary changes! Thank you